<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitBgDialog</class>
 <widget class="QDialog" name="FitBgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit Background without changing ratio</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="result_label">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>320</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="source_label">
           <property name="text">
            <string>Select Source Image Folder:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="source_btn">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="bg_label">
           <property name="text">
            <string>Select Background Image:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bg_btn">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="savepath_label">
           <property name="text">
            <string>Select Results Saving Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savepath_btn">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="color_label">
           <property name="text">
            <string>BG Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="colorRGBLabel">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>(255,255,255)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="colorDialogBtn">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="fit_btn">
           <property name="text">
            <string>Fit!</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
